tox.ini

```diff
@@ -323,3 +323,4 @@ commands =
     # Unsetting the TWINE_REPOSITORY_URL defaults twine to using production PyPI
     unset TWINE_REPOSITORY_URL
```

Comment:
If i understand the below change correctly, by setting `--repository pypi` we're explicitly telling it to go to prod pypi. In which case we can remove the (broken) `unset` line here.

Reply:
Update: I was wrong -- `TWINE_REPOSITORY_URL` overrides `--repository`, so we still want to unset it.

Reply:
https://github.com/aws/aws-encryption-sdk-cli/pull/185/commits/82a3dc65ece1768e05380d0bf0d9e02f74725ee2 :)

Being wrong is often the first step to learning something new :D